app/views/govuk_component/document_footer.raw.html.erb

```diff
@@ -30,9 +30,21 @@
       <% end %>
     <% end %>
     <% if history.any? %>
+      <% expand_full_history =
+          local_assigns.include?(:expand_full_history) &&
+          expand_full_history &&
+          history.size > 1
+      %>
+      <% if expand_full_history %>
+        <% rest_of_history = history[1..-1] %>
+        <% history = [history.first] %>
+      <% else %>
+        <% rest_of_history = [] %>
+      <% end %>
+
       <div class="change-notes" id="history">
         <h3>Page history</h3>
-        <ol>
+        <ol id="change-notes-history">
```

Comment:
We try avoid `id`s in components as it can create specificity problems, and issues with the the component guide where multiple instances of a component are rendered on the same page.

(_`id="history"` is an exception as it's used as an anchor link, though that should probably be configurable as an optional parameter_).

It looks like the new `id` is to support testing, could that be re-written as a more meaningful class instead?